### PR TITLE
[onnx-tools] dump graph and ops

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce onnx-dump and onnx-ops that dumps onnx model graph
and operators

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

File: compiler/onnx-tools/onnx-dump.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import onnx
+import os
+import subprocess
```
Comment (llFreetimell) [inline compiler/onnx-tools/onnx-dump.py:19]: Maybe not used?

File: compiler/onnx-tools/onnx-dump.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+            input_shape.append(dim.dim_value)
+        print('    I: {:<15} {} {}'.format(io_name, dtstr, input_shape))
+
+    for mod_output in onnx_model.graph.output:
+        io_name = '"{}"'.format(mod_output.name)
+        dtstr = _data_type_str(mod_output.type.tensor_type.elem_type)
+        shape = mod_output.type.tensor_type.shape
+        output_shape = []
+        for dim in shape.dim:
+            output_shape.append(dim.dim_value)
```
Comment (llFreetimell) [inline compiler/onnx-tools/onnx-dump.py:122]: (optional) Pythonic style :)
```suggestion
        output_shape = [dim.dim_value for dim in shape.dim]
```

**Comment by seanshpark** [comment]
@llFreetimell , thanks!

**Review by llFreetimell — APPROVED** [review]
LGTM :)

**Pull request merged**